components/claim/components/email_to_oauth.tsx

```diff
@@ -11,27 +10,41 @@ import * as Utils from 'utils/utils.jsx';
 import {t} from 'utils/i18n.jsx';
 import LoginMfa from 'components/login/login_mfa.jsx';
 import LocalizedInput from 'components/localized_input/localized_input';
+import { boolean } from '@storybook/addon-knobs';
```

Comment:
was this imported automatically by the IDE?

Reply:
Yes and I'm already deleting it